docs/docs/models/geneformer.md

```diff
@@ -166,19 +177,39 @@ This checkpoint was trained for approximately 11 epochs through the CELLxGENE sp
+The following describes the bert MLM token loss. Like in the original BERT paper, and the geneformer paper, 15% of all tokens are included in the loss. Of the included tokens, 80% are `"[MASK]"` token, 2% are a random gene token, and 18% are the correct output token. Note that this was an unintentional deviation from the original publication, but so far it seems to be working well. In the future we will test the intended 80%/10%/10% mixture proposed in the paper. The token loss in the following table is the mean cross entropy loss of the 15% of tokens included in the loss mask averaged across cells. As a baseline geneformer was downloaded from [the ctheodoris/Geneformer page on hugging face on 2024/11/04](https://huggingface.co/ctheodoris/Geneformer) and applied to the same masking/unmasking problem on this dataset, but with model-specific cell representations due to the updated tokenizer and medians dictionary used to train, and the update from training with 2048 tokens to 4096 tokens per cell. The held-out `test` dataset from our training splits described previously was used, and it should be noted that some of these cells may have been involved in training the baseline geneformer.
 
 | Model Description      | Token Loss (lower is better) |
 | ---------------------- | ---------------------------- |
-| Baseline geneformer    | 3.35                         |
-| geneformer-10M-240530  | 2.79                         |
-| geneformer-106M-240530 | 2.50                         |
+| Baseline geneformer    | 2.26*                        |
+| geneformer-10M-240530  | 2.64                         |
+| geneformer-106M-240530 | 2.34                         |
+
+!!! bug "Baseline Geneformer was recently updated on huggingface making loss comparisons challenging."
+
+    [Geneformer](https://huggingface.co/ctheodoris/Geneformer) was recently updated on hugging face to a new version.
+    In a future release we will make checkpoint conversion scripts available so that the public model can be ran
+    directly. Some key differences follow:
+
+     * Trained on a much larger 95M cell dataset. Our current checkpoints were trained with 23M cells.
+     * The new 12 layer baseline geneformer variant sits between our 10M and 106M parameter models in parameter count with
+      approximately 38M parameters.
+     * The model is trained with a 4096 context rather than a 2048 context. When forcing the model to make predictions
+      with a 2048 context, the MLM loss drops to *2.76*, which is probably unfair because this may be "out of domain" for
+      training. It is really hard to compare these loss numbers directly is the only take-home here.
+     * The model was trained on a set of 20275 genes, rather than the older set of 25426 genes. This would also be
+      expected to give a boost in loss since there are fewer tokens to chose from.
```

Comment:
```suggestion
     * The model was trained on a set of 20,275 genes, rather than the older set of 25,426 genes. This would also be
      expected to give a boost in loss since there are fewer tokens to choose from.
```